README.md

```diff
@@ -197,24 +197,16 @@ ibc-1:
         params:
           receive_enabled: true
           send_enabled: true
-relayer:
+relayer: # refer here (https://hermes.informal.systems/config.html) for more configs
   global:
-    timeout: 10s
-    light-cache-size: 20
-  paths:
```

Comment:
paths seem not used anywhere, I guess it's not needed for hermes? Have you tested it already?

Reply:
Yes, I have tested, `paths` is not used and needed for hermes.
Only configurations [here](https://hermes.informal.systems/config.html) are used by Hermes.